source/renderer.js

```diff
@@ -29,7 +30,5 @@ Renderer.prototype.error = function(command) {
 
 var audio = new Audio('assets/ding.mp3');
```

Comment:
you initialise the same class with the same name twice in the same file, it might be confusing